enterprise/cmd/frontend/internal/notebooks/resolvers/resolvers.go

```
@@ -356,6 +365,15 @@ func (r *notebookResolver) ViewerCanManage(ctx context.Context) bool {
 	return validateNotebookWritePermissionsForUser(r.notebook, actor.UID) == nil
 }
 
+func (r *notebookResolver) ViewerHasStarred(ctx context.Context) bool {
+	user, err := r.db.Users().GetByCurrentAuthUser(ctx)
+	if err != nil {
+		return false
```

Why not just return the error here? Seems like swallowing the error could make issues difficult to debug

---

I guess I could change the return signature to `(bool, error)` and return the error alongside the bool value?

---

I think that's probably better. The graphql layer will interpret that error nicely. We shouldn't ever have to call this manually, so the bonus error return value doesn't really add complexity